nussl/core/augmentation.py

```diff
@@ -0,0 +1,377 @@
+    Returns:
+        augmented_signal: A copy of the original audio signal, with augmentations applied 
+    """
+    if not np.issubdtype(type(lowest_freq), np.number) or lowest_freq <= 0:
+        raise ValueError("lowest_freq should be positve scalar")
+
+    h_stft = audio_signal.stft_data.copy()
+    freq_vector = audio_signal.freq_vector
+    idx = (np.abs(freq_vector - lowest_freq)).argmin()
+    h_stft[:idx, :, :] = 0
+    augmented_signal = audio_signal.make_copy_with_stft_data(h_stft)
+    augmented_signal.istft()
+
+    return augmented_signal
+
+def tremolo(audio_signal, mod_freq, mod_depth):
+    """
+    https://ffmpeg.org/ffmpeg-all.html#tremolo
+
+    Applies tremolo filter
+    Args: 
+        audio_signal: An AudioSignal object
+        mod_freq: Modulation frequency. Must be between 0 and 1.
+        mod_depth: Modulation depth. Must be between 0 and 1.
+    Returns:
+        augmented_signal: A copy of the original audio signal, with augmentations applied 
+    """
+    if not np.issubdtype(type(mod_freq), np.number) or mod_freq < 0:
+        raise ValueError("mod_freq should be positve scalar")
+    
+    if not np.issubdtype(type(mod_depth), np.number) or mod_depth < 0 or mod_depth > 1:
+        raise ValueError("mod_depth should be positve scalar between 0 and 1.")
+
+    filter_kwargs = {
+        "f": mod_freq,
+        "d": mod_depth
+    }
+
+    return apply_ffmpeg_filter(audio_signal, "tremolo", **filter_kwargs)
+
+def vibrato(audio_signal, mod_freq, mod_depth):
+    """
+    https://ffmpeg.org/ffmpeg-all.html#vibrato
+
+    Applies vibrato filter
+    Args: 
+        audio_signal: An AudioSignal object
+        mod_freq: Modulation frequency. Must be between 0 and 1.
+        mod_depth: Modulation depth. Must be between 0 and 1.
+    Returns:
+        augmented_signal: A copy of the original audio signal, with augmentations applied 
+    """
+    if not np.issubdtype(type(mod_freq), np.number) or mod_freq < 0:
+        raise ValueError("mod_freq should be positve scalar")
+    
+    if not np.issubdtype(type(mod_depth), np.number) or mod_depth < 0 or mod_depth > 1:
+        raise ValueError("mod_depth should be positve scalar between 0 and 1.")
+
+    filter_kwargs = {
+        "f": mod_freq,
+        "d": mod_depth
+    }
+
+    return apply_ffmpeg_filter(audio_signal, "vibrato", **filter_kwargs)
+
+def chorus(audio_signal, in_gain=.4, out_gain=.4, delays=[50], 
+    decays=[.8], speeds=[.95], depths=[.7]):
+    """
+    https://ffmpeg.org/ffmpeg-all.html#chorus
+
+    Applies Chorus Filter
+    Args:
+        audio_signal: An AudioSignal object
+        TODO: Write this documentation.
+    Returns:
+        augmented_signal: A copy of the original audio signal, with augmentations applied
+    """
+
+    # TODO: Arg checks
+    filter_kwargs = {
+        "in_gain": in_gain,
+        "out_gain": out_gain,
+        "delays": make_arglist_ffmpeg(delays),
```

Comment:
I think these utilities should go into this folder, instead of into `augmentation_utils.py`.